R/add_loop_healthcare_needed_cat.R

```diff
@@ -2,157 +2,138 @@
 #'
 #' @param loop A data frame of individual-level data.
 #' @param ind_healthcare_needed The name of the variable that indicates if healthcare is needed.
-#' @param ind_healthcare_needed_levels A character vector of levels for ind_healthcare_needed.
 #' @param ind_healthcare_received The name of the variable that indicates if healthcare is received.
-#' @param ind_healthcare_received_levels A character vector of levels for ind_healthcare_received.
+#' @param yes_value The value indicating "yes" for both needed and received variables.
+#' @param no_value The value indicating "no" for both needed and received variables.
+#' @param undefined_values A vector of values indicating undefined or unknown responses.
 #' @param wgq_dis The name of the variable that indicates if the individual has a disability (usual cut-offs are 3).
 #' @param ind_age The name of the variable that indicates the age of the individual.
 #'
 #' @export
 add_loop_healthcare_needed_cat <- function(
     loop,
     ind_healthcare_needed = "health_ind_healthcare_needed",
-    ind_healthcare_needed_levels = c("no", "yes", "dnk", "pnta"),
     ind_healthcare_received = "health_ind_healthcare_received",
-    ind_healthcare_received_levels = c("no", "yes", "dnk", "pnta"),
+    yes_value = "yes",
```

Comment:
Same proposal for consistency across the package to have here

yes_value --> ind_healthcare_needed_yes ...

Reply:
Thank you for reviewing! Suggested updates have been pushed. I will update the tests accordingly.